test/vast_parser.js

```diff
@@ -1210,6 +1210,68 @@ describe('VASTParser', function() {
     });
   });
 
+  describe('#Nester xml extensions', () => {
+    let ad;
+    before(done => {
+      vastParser
+        .getAndParseVAST(urlfor('nested-xml-extensions.xml'), {
```

Comment:
you should only have to call `parseExtensions` and not depend on `getAndParseVAST`. see example `spec/creative_companion_parser.spec.js` in https://github.com/dailymotion/vast-client-js/pull/280